Makefile

```diff
@@ -572,7 +576,7 @@ docker-build-data-image: docs-image
 
 .PHONY: docker-build-roxctl-image
 docker-build-roxctl-image:
-	cp -f bin/linux/roxctl image/roxctl/roxctl-linux
+	cp -f bin/linux_$(GOARCH)/roxctl image/roxctl/roxctl-linux
```

Comment:
This is referenced as `roxctl-linux-amd64` in `image/roxctl/Dockerfile`, l.4. Curious why this doesn't fail the build - don't we invoke this target?